designs/virtualization/4.x/boot source customization/boot-source-customization.md

```diff
@@ -0,0 +1,69 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# VM Templates - Boot Source Customization
+Updating the flow of the boot source customization.
+Removing the complexity of surfacing to the user the backend implementation of cloning the origin template first.
+Removing the option of resuming the customization process from the template clone to a notification at the top of the VM templates list.
+This PR will display the delta from the current implementation.
+
+## Entering the customization flow
+![Entering the flow](img/BS-00.jpg)
+The user can customize any available boot source for any template on the VM templates list.
+In order to enter it, the user can select "customize boot source" from the template actions menu,
+or by clicking the boot source itself and "customize" from within the popover.
+
+Note: a description is added to the boot source column title.
+
+![About this process modal](img/BS-01.jpg)
```

Comment:
Should we rephrase the last sentence to:
`The original template will be locked while the boot source is being cloned.`